contracts/QuestFactory.sol

```diff
@@ -388,6 +388,29 @@ contract QuestFactory is Initializable, LegacyStorage, OwnableRoles, IQuestFacto
         _claimCompressed(compressedData_, claimer);
     }
 
+    /// @dev Claim rewards for multiple quests on behalf of multiple claimers
+    function claimCompressedRefBatch(BatchClaimData[] calldata batchClaimDataArray) external payable {
+        uint256 totalFees = 0;
+        for (uint256 i = 0; i < batchClaimDataArray.length; i++) {
+            totalFees += batchClaimDataArray[i].fee;
+        }
```

Comment:
I think we can calculate totalFees by doing `batchClaimDataArray.length * mintFee`

Reply:
Doing this would also allow us to get rid of `fee` from the `BatchClaimData` struct